### DISCUSSION TITLE
How to test latest merged vanilla extract feature with remix

### DISCUSSION THREAD

**Discussion by MadadiSaeed** [remix-run/remix#5112] [Q&A] [answered]
Hi. I want to test and learn vanilla extract with remix. How to setup a remix project with vanilla extract?

**Comment by lili21** [comment, marked as answer]
You need to upgrade remix to the pre version.
then you can follow the [styling guide](https://github.com/remix-run/remix/blob/dev/docs/guides/styling.md#vanilla-extract).

**Reply by lili21** [reply]
the version is 1.11.0-pre.0

**Reply by MadadiSaeed** [reply]
Thanks for the help.
